python/social_bot/envs/gazebo_base.py

```diff
@@ -48,8 +51,45 @@ def render(self, mode='human'):
                         'GAZEBO_MASTER_URI'] = "localhost:%s" % self._port
                 self._rendering_process = Popen(['gzclient'])
             return
+        if mode == 'rgb_array':
+            if self._rendering_camera is None:
+                render_camera_sdf = """
+                <?xml version='1.0'?>
+                <sdf version ='1.4'>
+                <model name ='render_camera'>
+                    <static>1</static>
+                    <pose>-3 0 3 0 0.4 0</pose>
```

Comment:
A fixed pose probability does not fit for all different environment. We should allow different environment have different pose.

Reply:
Can you choose right pose for each of current environments?

Reply:
Sure.